_hacks/28.md

```diff
@@ -1,5 +1,23 @@
 ---
 hack_number: 28
 date: 2022-07-30
-upcoming: true
+upcoming: false
 ---
+
+A proper Summer hackday! People dipped in, people dipped out, Ryan painted a room, and we promised to record podcasts and then didn't. BUT that does set us up for some tasty mid-week SURPRISE podcasting sometime between now and the next hackday. Who knows what we might end up discussing?!?!?
+
+OH! And we actually had two people (Dan and Panda) actually FINISH SOMETHING! So it was notably one of our most successful hackdays evah!
+
+## Things people did
+
+- **Hugh** was thinking about soldering stuff, but thankfully, he was able to use his new concave half-of-a-keyboard to run calculations _mega-quickly_ to determine it would be cheaper to just start all over again. He also tidied his desk and then made everyone jealous by sharing a pic.
```

Comment:
```suggestion
- **Hugh** tidied his desk to safely solder stuff, but thankfully, he remembered he didn't have all the pieces and spent the day identifying all the missing pieces so that he can solder them another hack day. He also hacked the Irish government a bit more, making it just a little easier to find out what legislation they've been discussing and passing, using their API.
```